toast-tk-rest-utils/src/main/java/io/toast/tk/core/rest/RestUtils.java

```diff
@@ -158,49 +177,83 @@ public static ImportedScenarioDescriptor getScenario(final ImportedScenario scen
 		}
 	}
 
-	public static boolean post(final String url, final String jsonFixtureDescriptor) {
-		final Client httpClient = Client.create();
-		final WebResource webResource = httpClient.resource(url);
-		final ClientResponse response = webResource.type(MediaType.APPLICATION_JSON).accept(MediaType.APPLICATION_JSON)
-				.post(ClientResponse.class, jsonFixtureDescriptor);
-		return response.getStatus() == 200;
-	}
-
-	public static boolean postWebEventRecord(final String url, final String record) {
-		try {
-			final Client httpClient = Client.create();
+	public static boolean post(final String url, final String jsonFixtureDescriptor, final String ApiKey) {		
+		try(final CloseableHttpClient httpClient = HttpClients.createDefault()) {
+			if(Strings.isNullOrEmpty(ApiKey)) {
+				throw new IllegalAccessException("No Api Key provided !");

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "No Api Key provided !" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS1192)